### ISSUE TITLE
FORD support for Fortran documentation?

### ISSUE THREAD

**Issue by zbeekman** [LLNL/blt#90]
The [FORD] project is pretty awesome, and is a documentation solution built specifically for Fortran source code. It would be great to add support to blt for [FORD]. It can be configured in such a way as to be mostly comparable with Doxygen source file markup too.

Would you accept a PR implementing this, if I can find the time at some point? (Not likely to happen soon, though.)

Thanks!

[FORD]: https://github.com/cmacmackin/ford#readme

**Comment by cyrush** [comment]
@zbeekman, Interesting! Thanks for sharing. Yes, this is something we would be open to. 

I think we could easily treat it like Doxygen and Sphinx. (optional, supported in a build if FORD_EXECUTABLE is found)

It looks like FORD is a python package? So it should be easy for us to integrate into our CI testing.

**Comment by zbeekman** [comment]
Yes, FORD is available via PyPi (or whatever it's called now) and also
installable through the Homebrew package manager for macOS
